bin/list.js

```diff
@@ -1,5 +1,6 @@
 
 var soljsonSources = [
+  "soljson-v0.8.26+commit.8a97fa7a.js",
```

Comment:
This is weird, shouldn't it be a nightly like the previous ones?

Reply:
Ah never mind, it was like this in the previous binary pushes.

Reply:
I got a bit confused as well, but I looked at the previous PRs and they also update it. See: https://github.com/ethereum/solc-bin/pull/140/files and https://github.com/ethereum/solc-bin/pull/144/files